src/vmware/azext_vmware/_help.py

```diff
@@ -175,14 +195,33 @@
 
 helps['vmware datastore create'] = """
     type: command
-    short-summary: Create a datastore in a private cloud cluster.
+    short-summary: Please use "net-app-volume create" or "disk-pool-volume create" instead.
+"""
+
+helps['vmware datastore net-app-volume'] = """
+    type: group
+    short-summary: Create a new Microsoft.NetApp provided NetApp volume in a private cloud cluster.
+"""
+
+helps['vmware datastore net-app-volume create'] = """
+    type: command
+    short-summary: Create a new Microsoft.NetApp provided NetApp volume in a private cloud cluster.
     examples:
-    - name: Create a new Microsoft.StoragePool provided disk pool based iSCSI datastore.
-      text: az vmware datastore create --name iSCSIDatastore1 --resource-group MyResourceGroup --cluster Cluster-1 --private-cloud MyPrivateCloud --endpoints 10.10.0.1:3260 --lun-name lun0
-    - name: Create a new Microsoft.StoragePool provided disk pool based iSCSI datastore with multiple endpoints.
-      text: az vmware datastore create --name iSCSIDatastore1 --resource-group MyResourceGroup --cluster Cluster-1 --private-cloud MyPrivateCloud --endpoints 10.10.0.1:3260 10.10.0.2:3260 --lun-name lun0
     - name: Create a new Microsoft.NetApp provided NetApp volume based NFSv3 datastore.
-      text: az vmware datastore create --name ANFDatastore1 --resource-group MyResourceGroup --cluster Cluster-1 --private-cloud MyPrivateCloud --nfs-file-path ANFVol1FilePath --nfs-provider-ip 10.10.0.1
+      text: az vmware datastore net-app-volume create --name ANFDatastore1 --resource-group MyResourceGroup --cluster Cluster-1 --private-cloud MyPrivateCloud --volume-id /subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/ResourceGroup1/providers/Microsoft.NetApp/netAppAccounts/NetAppAccount1/capacityPools/CapacityPool1/volumes/NFSVol1
+"""
+
+helps['vmware datastore disk-pool-volume'] = """
+    type: group
+    short-summary: Create a new Microsoft.StoragePool provided disk pool based iSCSI datastore in a private cloud cluster.

```

Comment:
nit: Create a VMFS datastore in a private cloud cluster using Microsoft.StoragePool provided iSCSI target?

Reply:
Sure, in commit e56d2e1d227100c126d16c302598b960ce1cf938.